README.md

```diff
@@ -1,5 +1,16 @@
-dcrseeder
-=========
+# dcrseeder
+
+dcrseeder is a crawler for the Decred network, which exposes a list of reliable
+nodes via a built-in DNS server.
+
+When dcrseeder is started for the first time, it will connect to the dcrd node
+specified with the `-H` flag and listen for `addr` messages. These messages
```

Comment:
-s flag.   The seeder connects,  sends a getaddr request and waits for the addr message then disconnects.